src/Disks/DiskEncrypted.h

```diff
@@ -350,6 +350,11 @@ class DiskEncrypted : public IDisk
         return delegate;
     }
 
+    UInt32 getRefCount(const String & path) const override {
```

Comment:
Could you please add a test? Integration test, for example? See `ClickHouse/tests/integration`, there are several folders with tests for zero copy replication (contain "zero_copy" in folder name). You can add your test to one of the folder's `test.py` or add your own folder.